net-irc/dccserver/dccserver-0.5-r2.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs
+
+DESCRIPTION="linux implementation of the mirc dccserver command"
+HOMEPAGE="https://www.nih.at/dccserver/"
+SRC_URI="https://www.nih.at/dccserver/${P}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~riscv ~x86"
+
+RDEPEND="dev-libs/libbsd"
+DEPEND="${RDEPEND}"
+BDEPEND="${DEPEND}"
```

Comment:
I'll drop BDEPEND= on merge.

Reply:
Headers should always be in **DEPEND** and executable binaries in **BDEPEND** indeed.